source/SpinalHDL/Design errors/combinatorial_loop.rst

```diff
@@ -63,7 +63,7 @@ A fix could be :
 False-positive
 --------------
 
-It should be known that currently SpinalHDL is tracking combinatorial loop in a pessimistic way. If it give you a false positive combinatorial loop, you can manualy disable this checks on one signal of the loop.
+It should be said that SpinalHDL's algorithm to detect combinatorial loops can be pessimistic, and it may give false positives. If it is giving a false positive, you can manualy disable loop checking on one signal of the loop like so:
```

Comment:
(I had to google this) The examples I saw where it was used and wasn't used were all of the form "the table's leg" vs "leg of the table" or something similar. Applying this to the passage above, I think "SpinalHDL's algorithm" would become "the algorithm of SpinalHDL" which just doesn't sound right to me *shrug*. It may also be that American English (or vernacular anyway) is different than that in Europe in this case.

@wifasoi does "the algorithm of SpinalHDL" sound better to you?

Reply:
Sorry for the super late reply:
`the algorithm of SpinalHDL"` sound good to me